### DISCUSSION TITLE
Lag support for Linux hosts

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#1471] [Features]
Many production servers have redundant links with bonds on them. While it is good to keep the ```linux``` device simple, I would like to add lag support so I can use plain ```linux``` devices to model servers in our network

Any opinions/objections?

**Comment by ipspace** [comment]
Go for it, makes perfect sense.

**Comment by DanPartelly** [comment]
I like the idea. 

For years I have used link aggregations  on my linux laptop, so its not only useful on servers.  Its my preferred way  to handle Ethernet / wireless fail-overs. Everything is flawless as a single  IP is assigned on the bond interface.

**Comment by jbemmel** [comment]
Implemented in https://github.com/ipspace/netlab/pull/1624
